bindings/pyroot_experimental/PyROOT/test/tcomplex_radd_rsub.py

```diff
@@ -0,0 +1,43 @@
+import unittest
+import ROOT
+from ROOT import TComplex
+
+class TestTComplexMethods(unittest.TestCase):
+        """
+        Test for the operators of TComplex:
+        __add__, __sub__, __radd__, __rsub__.
+        """
+
+        c1 = TComplex(4.,0)
+        c2 = TComplex(1.,1)
+
+        d = 2.
+
+        # check if both __add__ and __radd__ give the expected result with complex + double and double + complex and if __radd__ is equal to __add__ both for complex + complex and complex + double (real components)
+        def test_radd(self):
+                self.assertEqual((self.c1 + self.d).Re(), 6.0)
+                self.assertEqual((self.d + self.c1).Re(), 6.0)
+                self.assertEqual((self.c1 + self.c2).Re(), (self.c2 + self.c1).Re())
```

Comment:
This one should not invoke `__radd__` in any of the operands you are comparing, since it can invoke the left addition on both `c1` for the first operand and `c2` for the second. This one should also work without the pythonisation (this highlights the importance of first writing and running the tests, to check what is failing).